package.json

```diff
@@ -83,6 +83,6 @@
     "shelljs": "^0.8.4",
     "ts-node": "^8.10.2",
     "tslint": "^6.1.3",
-    "typescript": "4.0.2"
+    "typescript": "4.1.2"
```

Comment:
`ng update` doesn't bump this, but I saw that it's supported. I guess we'll try it out and see if there are any issues.

Reply:
I bumped it since it's the version we're using on the main repo.